all_robots.xml

```diff
@@ -69,4 +69,5 @@
 	<xi:include href="./iCubValparaiso01/icub_all.xml" />
 	<xi:include href="./ergoCubSN000/ergocub_all.xml" />
 	<xi:include href="./iCubShanghai01/icub_all.xml" />
+	<xi:include href="./ergoCubSN001/ergocub_all.xml" />
```

Comment:
Would you mind inserting the new line right after ergoCubSN000?
This will help later on.
Thanks!

Reply:
@pattacini can I move `iCubShanghai01` above and keep ergoCub000 and 001 as the last in the list?

Reply:
Yes!